<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ManageGridDialog</class>
 <widget class="QDialog" name="ManageGridDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>812</width>
    <height>160</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>sconcho: Manage Grid</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>insert row</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>&amp;insert</string>
           </property>
           <property name="buddy">
            <cstring>numInsertRows</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numInsertRows">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_2">
           <property name="text">
            <string>row&amp;s</string>
           </property>
           <property name="buddy">
            <cstring>insertRowMode</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="insertRowMode">
           <item>
            <property name="text">
             <string>below</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>above</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_3">
           <property name="text">
            <string>&amp;row</string>
           </property>
           <property name="buddy">
            <cstring>insertRowPivot</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="insertRowPivot">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="insertRowButton">
           <property name="text">
            <string>Insert Row(s)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>delete row</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_5">
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>&amp;delete</string>
           </property>
           <property name="buddy">
            <cstring>numDeleteRows</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numDeleteRows">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>row&amp;s</string>
           </property>
           <property name="buddy">
            <cstring>insertRowMode</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="deleteRowMode">
           <item>
            <property name="text">
             <string>at                          </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>above</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>&amp;row</string>
           </property>
           <property name="buddy">
            <cstring>insertRowPivot</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="deleteRowPivot">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="deleteRowButton">
           <property name="text">
            <string>Delete Row(s)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>insert column</string>
      </attribute>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>&amp;insert</string>
           </property>
           <property name="buddy">
            <cstring>numInsertColumns</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numInsertColumns">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>column&amp;s</string>
           </property>
           <property name="buddy">
            <cstring>insertColumnMode</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="insertColumnMode">
           <item>
            <property name="text">
             <string>left of</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>right of</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>&amp;column</string>
           </property>
           <property name="buddy">
            <cstring>insertColumnPivot</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="insertColumnPivot">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>40</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="insertColumnButton">
           <property name="text">
            <string>Insert Column(s)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_4">
      <attribute name="title">
       <string>delete column</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>&amp;delete</string>
           </property>
           <property name="buddy">
            <cstring>numDeleteColumns</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="numDeleteColumns">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>column&amp;s</string>
           </property>
           <property name="buddy">
            <cstring>insertRowMode</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="deleteColumnMode">
           <item>
            <property name="text">
             <string>at                                      </string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>left of</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>&amp;column</string>
           </property>
           <property name="buddy">
            <cstring>insertRowPivot</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="deleteColumnPivot">
           <property name="minimum">
            <number>1</number>
           </property>
           <property name="maximum">
            <number>1000</number>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="horizontalSpacer_4">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>38</width>
             <height>20</height>
            </size>
           </property>
          </spacer>
         </item>
         <item>
          <widget class="QPushButton" name="deleteColumnButton">
           <property name="text">
            <string>Delete Column(s)</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>644</width>
       <height>14</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_9">
     <item>
      <spacer name="horizontalSpacer_5">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="closeButton">
       <property name="text">
        <string>Close</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>tabWidget</tabstop>
  <tabstop>numInsertRows</tabstop>
  <tabstop>insertRowMode</tabstop>
  <tabstop>insertRowPivot</tabstop>
  <tabstop>insertRowButton</tabstop>
  <tabstop>numDeleteRows</tabstop>
  <tabstop>deleteRowButton</tabstop>
  <tabstop>numInsertColumns</tabstop>
  <tabstop>insertColumnMode</tabstop>
  <tabstop>insertColumnPivot</tabstop>
  <tabstop>insertColumnButton</tabstop>
  <tabstop>numDeleteColumns</tabstop>
  <tabstop>deleteColumnButton</tabstop>
  <tabstop>closeButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
